pkgs/phps.nix

```diff
@@ -43,9 +54,21 @@ let
 
               "--enable-libxml"
               "--with-libxml-dir=${prev.libxml2.dev}"
+              (
+                if prev.lib.versionAtLeast args.version "7.3"
+                then "--with-pcre-regex=${prev.pcre2.dev}"
+                else "--with-pcre-regex=${prev.pcre.dev}"
+              )
```

Comment:
This is now redundant with the lines below.

Reply:
Not anymore :)